compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/controlcode_to_transaction.mlir

```diff
@@ -68,10 +68,10 @@ module attributes {hal.executable.target = #executable_target_amdaie_xclbin_fb}
 // -----
 
 // CHECK:       0x06030100
-// CHECK:       0x00000105
+// CHECK:       0x00000104
 // CHECK:       0x00000001
 // CHECK:       0x00000028
-// CHECK:       0x00000000
+// CHECK:       0x00140000
```

Comment:
This is a workaround to accommodate `aie-rt`.

I believe the following implementation of `_XAie_GetRowfromRegOff` is actually wrong, as it behaves incorrectly by always preserving the lowest 8 bits.
https://github.com/Xilinx/aie-rt/blob/1470f6c1d0e765a73c741ad51bc761515d74457e/driver/src/common/xaie_helper.c#L1024-L1027

In contrast, `_XAie_GetColfromRegOff` appears to be correctly implemented and performs as expected.

Reply:
Where exactly do you perform this workaround?

Reply:
I mean the correct value should be `0x00000000` by definition (as it operates on row 0, col 0). However, because the current aie-rt implementation behaves differently, the CHECK value is updated accordingly. This discrepancy currently does not cause any hardware errors, as these values are discarded later in the process. Nonetheless, I will report this issue to the aie-rt repository.

Reply:
Ok, I see, thanks for the info, would indeed be good to report this.